wire-compiler/src/main/java/com/squareup/wire/WireCompiler.kt

```diff
@@ -372,6 +381,11 @@ class WireCompiler internal constructor(
             schemaHandlerFactoryClass = arg.substring(SCHEMA_HANDLER_FACTORY_CLASS_FLAG.length)
           }
 
+          arg.startsWith(CUSTOM_OPTION_FLAG) -> {
+            val (key, value) = arg.substring(SCHEMA_HANDLER_FACTORY_CLASS_FLAG.length).split(',')
+            customOptions[key] = value
```

Comment:
@JakeWharton made it simpler

Reply:
This is different that what you asked, though

Reply:
Yep yep I made the API simpler. I was going with `options=key,value[,key,value]`